arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,6 +410,11 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current state of Analysis Services resource. It allows more custom terminal state like Paused."
```

Comment:
Yes, state like paused would be the state outside of resource provisioning.
In this case, ProvisioningState='Succeeded', State='Paused'

@ravbhatnagar 
I think it's better to be consistent with name from response in ARM-AnalysisService
Can we keep state unless service side change state to status?